SignalServiceKit/src/Messages/Interactions/TSInfoMessage.m

```diff
@@ -100,7 +117,14 @@ - (NSString *)previewTextWithTransaction:(YapDatabaseReadTransaction *)transacti
         case TSInfoMessageTypeUnsupportedMessage:
             return NSLocalizedString(@"UNSUPPORTED_ATTACHMENT", nil);
         case TSInfoMessageUserNotRegistered:
-            return NSLocalizedString(@"CONTACT_DETAIL_COMM_TYPE_INSECURE", nil);
+            if (self.unregisteredRecipientId.length > 0) {
+                return [NSString stringWithFormat:NSLocalizedString(@"ERROR_UNREGISTERED_USER_FORMAT",
+                                                      @"Format string for 'unregistered user' error. Embeds {{the "
+                                                      @"unregistered user's signal id}}."),
+                                 self.unregisteredRecipientId];
```

Comment:
Why not use the display name?

Reply:
Done.

On a side note, I'd like to eventually refactor `previewTextWithTransaction` to take a contact manager param.